server/api/controllers/forms/post-delete-form.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  friendlyName: "Delete form by id",
+
+  description: "Delete form by id",
+
+  inputs: {
+    id: {
+      type: "string",
+      required: true,
+      description: "Id of form",
+    },
+  },
+
+  exits: {},
+
+  fn: async function ({ id }, exits) {
+    try {
+      const data = await Form.destroy(id);
```

Comment:
i'm assuming this already handles if id doesn't exist? if not, then will this already handle that situation?

Reply:
also, should probably consider using soft-delete